ggshield/docker.py

```diff
@@ -50,12 +51,13 @@ def docker_save_to_tmp(image_name: str, temporary_path: str) -> Path:
     command = ["docker", "save", image_name, "-o", str(temp_archive_filename)]
 
     try:
-        subprocess.run(
-            command,
-            check=True,
-            stderr=subprocess.PIPE,
-            timeout=DOCKER_COMMAND_TIMEOUT,
-        )
+        with yaspin(text="Saving docker image"):
+            subprocess.run(
+                command,
+                check=True,
+                stderr=subprocess.PIPE,
+                timeout=DOCKER_COMMAND_TIMEOUT,
+            )
```

Comment:
If we want to persist "Saving docker image" on output and add a "ok" once done, we can do:
```
with yaspin(text="Saving docker image") as spinner:
    subprocess.xxx
    spinner.ok()
```
I preferred to make this text disappear once done, but you can try it if you want.